contact_us.html

```diff
@@ -273,10 +273,14 @@ <h1>Zachary Jeckell</h1>
       </li>
         <li class="hex">
         <div class="hexIn">
-          <a class="hexLink" href="{insert link here}">
+          <a class="hexLink" href="{https://www.linkedin.com/in/grzegorz-golba-9a21341a2/}">
             <img src="img/members/grzegorz_golba.jpg" alt="Grzegorz Golba" />
```

Comment:
```suggestion
            <img src="img/members/grzegorz_golba.png" alt="Grzegorz Golba" />         
```
If you execute ``bundle exec jekyll serve`` in the website directory you can see the local changes (i.e. on your computer) to the site without publishing them, just in case something is broken. In this case your picture will not show up because it points to a file that doesn't exist ``grzegroz_golba.png`` != ``grzegorz_golba.jpg``. You need to either change the text in the ``contact_us.html`` or change the file extension. I recommend the former.